src/web/handlers.rs

```diff
@@ -1,15 +1,28 @@
 //! API Handlers
 use actix_web::{web, HttpRequest, HttpResponse};
+use rand::{thread_rng, Rng};
 
 use crate::{
     adm,
     error::{HandlerError, HandlerErrorKind},
     metrics::Metrics,
     server::{cache, location::LocationResult, ServerState},
+    settings::Settings,
     tags::Tags,
     web::{middleware::sentry as l_sentry, DeviceInfo},
 };
 
+/// Calculate the ttl from the settings by taking the tiles_ttl
+/// and calculating a jitter that is no more than 50% of the total TTL.
+/// It is recommended that "jitter" be 10%.
+pub fn add_jitter(settings: &Settings) -> u32 {
+    let mut rng = thread_rng();
+    let ftl = settings.tiles_ttl as f32;
+    let offset = ftl * (std::cmp::min(settings.jitter, 50) as f32 * 0.01);
+    let jit = rng.gen_range(0.0 - offset..offset);
```

Comment:
Shouldn't it be inclusive (I don't actually care 😄 )?

```suggestion
    let jit = rng.gen_range(0.0 - offset..=offset);
```